features/step_definitions/basic_steps.rb

```diff
@@ -24,7 +24,7 @@
 
 def page_path_from(page_name)
   case page_name.downcase
-  when 'registration'
+  when "registration"
```

Comment:
Are you kidding me? Are you actually changing back to double quotes?

Reply:
we were told to change all the single quotes to double quotes in our conflicted pull request we closed, does this go for all the quotes in the file ?